### DISCUSSION TITLE
Using BlenderBot 2.0 for other languages

### DISCUSSION THREAD

**Discussion by gianlucabusatta** [facebookresearch/ParlAI#3919] [Q&A]
(I'm opening this discussion in parallel with [this one](https://github.com/facebookresearch/ParlAI/discussions/3633) about BlenderBot 1)

Is there any work which already uses BlenderBot 2.0 in another language?

For instance, I'm interessed in using it in Italian. How should I do it?

- Fine-tuning on a dataset with the language of my interest?
- Change some BlenderBot 2.0 underlying model? Which one? How it can be done? Is there any tutorial to do it?

Thank you in advance for the help.

**Comment by stephenroller** [comment]
I don't know of any one working on non-English BB2

**Reply by gianlucabusatta** [reply]
Thank you for the answer.

I have further questions:

1. Let's suppose to use a Neural Machine Translation model to translate the input and the output and use BB2 as it is. Is there any clue about how well BB2 would works?

2. Let's suppose I would like to change some model inside BB2 with its multilingual variant; which parts are strictly necessary to be changed?

**Reply by stephenroller** [reply]
These are fine research questions that I don't have any evidence to answer with.
